README.md

```
@@ -138,7 +138,7 @@ suspend fun executePrompt() {
 }
 ```
 
-See the [complete example](examples/src/main/kotlin/me/kpavlov/koog/sauce/examples/LangChain4jAIAgentExample.kt).   
+See the [complete example](examples/jvm/src/main/kotlin/me/kpavlov/koog/sauce/examples/LangChain4jAIAgentExample.kt).   
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)